src/strategies/honeyswap/index.ts

```diff
@@ -0,0 +1,81 @@
+import { getAddress } from '@ethersproject/address;
```

Comment:
Hi @kittyslasher, missed a quote here :) 
```
src/strategies/honeyswap/index.ts(1,52): error TS1002: Unterminated string literal.
```
```suggestion
import { getAddress } from '@ethersproject/address';
```